packages/ecs/src/schemas/JSONSchemas.ts

```diff
@@ -279,5 +279,36 @@ export const S = {
     ({
       ...buildSchema('Proxy', { create: proxy, serialized: true }),
       properties: schema
-    }) as TProxySchema<T>
+    }) as TProxySchema<T>,
+
+  GLTFExtensionSchema: <T extends Schema>(schema: T): T => {
```

Comment:
i don't think we need this, the specific component serializers/deserializers alone should suffice. i don't anticipate we'll have many more than uuid and visible, virtually everything else will be an object